### DISCUSSION TITLE
How to create multiple sites in a single bench and use all at the same time?

### DISCUSSION THREAD

**Discussion by crelocks** [rtCamp/Frappe-Manager#204] [Q&A]
How do you create multiple sites in a single bench and use them all simultaneously?
fm version =  0.14.0

frappe  version = 15.31.0

**Comment by Xieyt** [comment]
Creating multiple sites in a bench is not supported at the moment. I have create an issue regarding this [here](https://github.com/rtCamp/Frappe-Manager/issues/207). You can keep track of the feature release by following the issue.
